CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Securit
 
 ## Unreleased
 
+### Changed
+
+* Optimizations in Nextflow Pipeline Applet script to make fewer API calls when
+concluding a subjob
+
+### Fixed
+
+* If Nextflow subjob finished with error and error strategy was terminate, wait up to 240s
+then exit the subjob with error, which will cause the pipeline exec tree to fail
+(consistent with Nextflow's expected behavior if not timed out).
```

Comment:
It is not true that this is consistent. It's just inconsistent in a different manner. I've added comment to the jira ticket mentioned below.